Docker.DotNet/ContainerOperations.cs

```diff
@@ -134,6 +134,23 @@ public Task<Stream> ExportContainerAsync(string id, CancellationToken cancellati
             return response.StatusCode != HttpStatusCode.NotModified;
         }
 
+        public async Task<ExecContainerResponse> ExecContainerAsync(string id, Config config)
+        {
+            if (string.IsNullOrEmpty(id))
+            {
+                throw new ArgumentNullException("id");
+            }
+
+            string path = string.Format(CultureInfo.InvariantCulture, "containers/{0}/exec", id);
+            JsonRequestContent<Config> data = null;
```

Comment:
also `/exec` does not take a Config object... It accepts only a small number of fields:

http://docs.docker.com/reference/api/docker_remote_api_v1.20/#exec-create

Please create a `ExecContainerParameters` object with these fields.